service/matching/config.go

```diff
@@ -49,6 +49,7 @@ type (
 		TestDisableSyncMatch                  dynamicconfig.BoolPropertyFn
 		RPS                                   dynamicconfig.IntPropertyFn
 		OperatorRPSRatio                      dynamicconfig.FloatPropertyFn
+		AlignMembershipChange                 dynamicconfig.DurationPropertyFn
```

Comment:
Could you help me better understand what this property mean? AlignMembershipChange is time we wait for membership to stabilize after change, is this correct?

Reply:
no, it's the time interval that we try to align different membership changes to. i.e. if one one started at 3:15:03 and another one stopped at 3:15:05, and a third started at 3:15:06.5, we will make all three changes effective as of 3:15:10